index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+	<head>
+		<meta charset="utf-8">
+		<title>Apple</title>
+
+		<link rel="stylesheet" type="text/css" href="css/reset.css">
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+	</head>
+	<body>
+		<main class="main-container">
+			<div class="container">
+				<nav class="nav-menu">
+					<ul class="menu">
+						<li class="menu-link" id="logo"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/"><img src="img/apple.png" width="20" class="logo" alt="Apple"></a></li>
+						<li class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://store.apple.com/">Store</a></li>
+						<li class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/mac/">Mac</a></li>
+						<li class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/">iPod</a></li>
+						<li class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/">iPhone</a></li>
+						<li class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/">iPad</a></li>
+						<li class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/">iTunes</a></li>
+						<li class="menu-link"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/">Support</a></li>
+						<li class="menu-link" id="button"><input class="search" type="search" name="search"></li>
+					</ul>
+				</nav>
+				<a class="main-img" href="https://web.archive.org/web/20140301004610/http:/www.apple.com/your-verse/"></a>
+				<div class="div-text">
+					<a class="text" href="https://web.archive.org/web/20140301004610/http:/www.apple.com/your-verse/"></a>
+					<a class="text2" href="https://web.archive.org/web/20140301004610/http:/www.apple.com/your-verse/"></a>
+					<a class="text3" href="https://web.archive.org/web/20140228175622/http://www.apple.com/your-verse/#video-your-verse"></a>
+				</div>
+			</div>
+		</main>
+		<footer>
+			<div class="footer-imgs">
+				<a class="footer-img-1" href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone-5s/"></a>
```

Comment:
For this section, the top part is made of four `<a>` tags with `<div>` tags inside it.
The `<div>` tag inside the `<a>` tag should use an image as background